source/vibe/core/sync.d

```diff
@@ -354,6 +354,7 @@ private void runMutexUnitTests(M)()
 		assert(m.m_impl.m_locked);
 		t1.join();
 		assert(!t1.running && t2.running);
+		yield();
```

Comment:
Can you add a comment akin to the one for the other yield here? `// give t2 the chance to acquire the lock`